app/controllers/comments_controller.rb

```diff
@@ -230,6 +230,22 @@ def unhide
     end
   end
 
+  def admin_delete
+    @comment = Comment.find(params[:comment_id])
+    authorize @comment
+    @comment.deleted = true
+
+    if @comment.save
+      redirect_url = @comment.commentable&.path
+      if redirect
```

Comment:
I think you meant `redirect_url` here. Also, what is the fallback if redirect_url is nil?

Reply:
Good point Molly, will address 😄